docs/contributing/code_of_conduct.rst

```diff
@@ -0,0 +1,232 @@
+-------------------------------------
+
+You will receive an email from the Code of Conduct committee
+acknowledging receipt immediately. We promise to acknowledge receipt
+within 24 hours (and will aim for much quicker than that).
+
+The committee will immediately meet to review the incident and
+determine:
+
+#. What happened.
+#. Whether this event constitutes a code of conduct violation.
+#. Who the bad actor was.
+#. Whether this is an ongoing situation, or if there is a threat to
+   anyone's physical safety.
+
+If this is determined to be an ongoing incident or a threat to physical
+safety, the committee's immediate priority will be to protect
+everyone involved. This means we may delay an "official" response until
+we believe that the situation has ended and that everyone is physically
+safe.
+
+Once the committee has a complete account of the events they will
+make a decision as to how to response. Responses may include:
+
+ * Nothing (if we determine no violation occurred).
+ * A private reprimand from the committee to the individual(s)
+   involved.
+ * A public reprimand.
+ * An imposed vacation (i.e. asking someone to "take a week off" from a
+   mailing list or Slack).
+ * A permanent or temporary ban from some or all communication spaces (mailing
+   lists, Slack, etc.)
+ * A request for a public or private apology.
+
+We'll respond within one week to the person who filed the report with
+either a resolution or an explanation of why the situation is not yet
+resolved.
+
+Once we've determined our final action, we'll contact the original
+reporter to let them know what action (if any) we'll be taking. We'll
+take into account feedback from the reporter on the appropriateness of
+our response, but we don't guarantee we'll act on it.
+
+Attribution
+-----------
+
+This Code of Conduct is distributed under a `Creative Commons
+Attribution-ShareAlike
+license <http://creativecommons.org/licenses/by-sa/3.0/>`__.
+
+Text derived from the
+`Django Code of Conduct <https://www.djangoproject.com/conduct/reporting/>`__
+
+
+Enforcement Manual
+==================
+
+This is the enforcement manual followed by Learning Equality's Code of Conduct
+Committee. It's used when we respond to an issue to make sure we're
+consistent and fair. It should be considered an internal document, but
+we're publishing it publicly in the interests of transparency.
+
+The Code of Conduct Committee
+-----------------------------
+
+All responses to reports of conduct violations will be managed by a Code
+of Conduct Committee ("the committee").
+
+Learning Equality's (LE's) core team ("the core") will
+establish this committee, comprised of at least three members.
+
+
+How the committee will respond to reports
+-----------------------------------------
+
+When a report is sent to the committee they will immediately reply to
+the report to confirm receipt. This reply must be sent within 24 hours,
+and the committee should strive to respond much quicker than that.
```

Comment:
I'm not arguing for it, but since "immediately" is not "within 24 hours", let's just say the committee will reply to confirm receipt within 24 hours, and drop the sentence with "immediately".